impeller/display_list/display_list_dispatcher.cc

```diff
@@ -862,6 +884,8 @@ void DisplayListDispatcher::drawImageLattice(
     flutter::DlFilterMode filter,
     bool render_with_attributes) {
   // Needs https://github.com/flutter/flutter/issues/95434
+  // Don't implement this one since it is not exposed by flutter,
```

Comment:
A note for myself since I actually spent some time searching for usages of this in the engine.

Reply:
More importantly, the DLCanvasRecorder converts the calls back to drawImageNine after Skia converted them to a lattice...

Reply:
Added that detail too...